### PR TITLE
Animals table

### PR DESCRIPTION
**Pull request by hafiz1379**
### `Vet Clinic Database`

Hello 🙋‍♂️🙋‍♂️
🔰 In this branch I:
✅ Use [this template](https://github.com/microverseinc/curriculum-template-databases) to generate my repo.
✅ Create a database named vet_clinic.
✅ Create a table named animals with the following columns:

- id: integer
- name: string
- date_of_birth: date
- escape_attempts: integer
- neutered: boolean
- weight_kg: decimal

✅ Copy the SQL I used in the previous point into the schema.sql file in the generated repository.
✅ Insert the following data:

- Animal: His name is Agumon. He was born on Feb 3rd, 2020, and currently weighs 10.23kg. He was neutered and he has never tried to escape.
- Animal: Her name is Gabumon. She was born on Nov 15th, 2018, and currently weighs 8kg. She is neutered and she has tried to escape 2 times.
- Animal: His name is Pikachu. He was born on Jan 7th, 2021, and currently weighs 15.04kg. He was not neutered and he has tried to escape once.
- Animal: Her name is Devimon. She was born on May 12th, 2017, and currently weighs 11kg. She is neutered and she has tried to escape 5 times.
- Copy the SQL you used in the previous point into the data.sql file in the generated repository.
- 
✅ Write queries for the following quests:

1. Find all animals whose name ends in "mon".
image

2. List the name of all animals born between 2016 and 2019.
image

3. List the name of all animals that are neutered and have less than 3 escape attempts.
image

4. List the date of birth of all animals named either "Agumon" or "Pikachu".
image

5. List name and escape attempts of animals that weigh more than 10.5kg.
image

6. Find all animals that are neutered.
image

7. Find all animals not named Gabumon.
image

8. Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg).
image

✅ Take a screenshot of the results of my queries.
✅ Copy the SQL I used in the previous point into the queries.sql file in the generated repository.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,196 @@
...
+
+I would like to thank my mentor and my cod partners for support
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (ShadyShawkat) [inline README.md]: - [ ] Please make sure that this link works as intended and opens up the MIT license.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @hafiz1379, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- All linters are passing ✔️ 
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Munsa1 — APPROVED** [review]
Hi @hafiz1379, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Fixed Broken Link :heavy_check_mark: 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by hafiz1379** [comment]
@Munsa1 Thank you so much for your thorough review and for approving the pull request!

**Pull request merged**